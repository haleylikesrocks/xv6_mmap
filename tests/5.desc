File Backed mmap test : Accessing mmap-ed memory region after file close.
The contents should persist.
