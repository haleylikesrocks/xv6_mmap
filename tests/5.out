XV6_TEST_OUTPUT : file open suceeded
XV6_TEST_OUTPUT : First Read returned : 13
XV6_TEST_OUTPUT : file content : Hello World.!
XV6_TEST_OUTPUT : file close suceeded
XV6_TEST_OUTPUT : file open suceeded
XV6_TEST_OUTPUT : mmap suceeded
XV6_TEST_OUTPUT : file close suceeded
XV6_TEST_OUTPUT : File content from mmap: Hello World.!
XV6_TEST_OUTPUT : Return val of strcmp 0
XV6_TEST_OUTPUT : File content and memory mapped content are Same.
XV6_TEST_OUTPUT : munmap suceeded
